components/widgets/menu/menu_items/menu_item_action.tsx

```diff
@@ -42,23 +42,7 @@ export const MenuItemActionImpl = ({
         }
         onClick={onClick}
     >
-        {text && (
```

Comment:
I wlll put this here since this not a change in this PR, but it should be changed nonetheless:

In lines 33-41 the `classNames` function is used incorrectly. Could you please replace it with this:

```typescript
className={classNames('style--none', buttonClass, {
    'MenuItem__with-help': extraText,
    MenuItem__dangerous: isDangerous,
})}
```

there is no need to add `buttonClass` as an object property, since any falsy values will not make it in the classList when using it inside the array (or even in the params list) of the `classNames` function